### ISSUE TITLE
api_app_job"."errors", "api_app_job"."file"

### ISSUE THREAD

**Issue by ibrahim3pf** [intelowlproject/IntelOwl#112]
Hello,
Iam tryign to install the tool but would never finish the instalation via docker,
I did the following:

1- cp env_file_app_template env_file_app
2- cp env_file_postgres_template env_file_postgres
3- docker-compose up

but docker still stuck in here:

intel_owl_celery_worker | [2020-07-10 22:00:01,282: INFO/ForkPoolWorker-2] started check_stuck_analysis
intel_owl_postgres | 2020-07-10 22:00:01.293 UTC [68] ERROR:  relation "api_app_job" does not exist at character 616
intel_owl_postgres | 2020-07-10 22:00:01.293 UTC [68] STATEMENT:  SELECT "api_app_job"."id", "api_app_job"."source", "api_app_job"."is_sample", "api_app_job"."md5", "api_app_job"."observable_name", "api_app_job"."observable_classification", "api_app_job"."file_name", "api_app_job"."file_mimetype", "api_app_job"."status", "api_app_job"."analyzers_requested", "api_app_job"."run_all_available_analyzers", "api_app_job"."analyzers_to_execute", "api_app_job"."analysis_reports", "api_app_job"."received_request_time", "api_app_job"."finished_analysis_time", "api_app_job"."force_privacy", "api_app_job"."disable_external_analyzers", "api_app_job"."errors", "api_app_job"."file" FROM "api_app_job" WHERE "api_app_job"."status" = 'running'
intel_owl_celery_worker | [2020-07-10 22:00:01,295: ERROR/ForkPoolWorker-2] Task intel_owl.tasks.check_stuck_analysis[90719575-7175-4053-b758-49527d65ea30] raised unexpected: ProgrammingError('relation "api_app_job" does not exist\nLINE 1: ..."api_app_job"."errors", "api_app_job"."file" FROM "api_app_j...\n                                                             ^\n',)
intel_owl_celery_worker | Traceback (most recent call last):
intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
intel_owl_celery_worker |     return self.cursor.execute(sql, params)
intel_owl_celery_worker | psycopg2.errors.UndefinedTable: relation "api_app_job" does not exist
intel_owl_celery_worker | LINE 1: ..."api_app_job"."errors", "api_app_job"."file" FROM "api_app_j...
intel_owl_celery_worker |                                                              ^
intel_owl_celery_worker | 
intel_owl_celery_worker | 
intel_owl_celery_worker | The above exception was the direct cause of the following exception:
intel_owl_celery_worker | 
intel_owl_celery_worker | Traceback (most recent call last):
intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/celery/app/trace.py", line 412, in trace_task
intel_owl_celery_worker |     R = retval = fun(*args, **kwargs)
intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/celery/app/trace.py", line 704, in __protected_call__
intel_owl_celery_worker |     return self.run(*args, **kwargs)
intel_owl_celery_worker |   File "/opt/deploy/intel_owl/intel_owl/tasks.py", line 61, in check_stuck_analysis
intel_owl_celery_worker |     crons.check_stuck_analysis()
intel_owl_celery_worker |   File "/opt/deploy/intel_owl/api_app/crons.py", line 22, in check_stuck_analysis
intel_owl_celery_worker |     logger.info(f"checking if {len(running_jobs)} jobs are stuck")
intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/django/db/models/query.py", line 256, in __len__
intel_owl_celery_worker |     self._fetch_all()
intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/django/db/models/query.py", line 1242, in _fetch_all
intel_owl_celery_worker |     self._result_cache = list(self._iterable_class(self))
intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/django/db/models/query.py", line 55, in __iter__
intel_owl_celery_worker |     results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1140, in execute_sql
intel_owl_celery_worker |     cursor.execute(sql, params)
intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 67, in execute
intel_owl_celery_worker |     return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 76, in _execute_with_wrappers
intel_owl_celery_worker |     return executor(sql, params, many, context)
intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
intel_owl_celery_worker |     return self.cursor.execute(sql, params)
intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/django/db/utils.py", line 89, in __exit__
intel_owl_celery_worker |     raise dj_exc_value.with_traceback(traceback) from exc_value
intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
intel_owl_celery_worker |     return self.cursor.execute(sql, params)
intel_owl_celery_worker | django.db.utils.ProgrammingError: relation "api_app_job" does not exist
intel_owl_celery_worker | LINE 1: ..."api_app_job"."errors", "api_app_job"."file" FROM "api_app_j...

Thanks.

**Comment by Eshaan7** [comment]
Hi. It looks like the tables didn't got created in the database for some reason.
With `docker-compose up` running, open another terminal and run this command:
```
docker exec -ti intel_owl_uwsgi bash
```
Once inside the docker container run these 2 commands, 
```
python manage.py makemigrations api_app
python manage.py migrate api_app
```

**Comment by ibrahim3pf** [comment]
Thanks Eshaan7

I recieve this after excuting the commands:

> intel_owl_celery_worker |   File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
intel_owl_celery_worker |     return self.cursor.execute(sql, params)
intel_owl_celery_worker | django.db.utils.ProgrammingError: relation "token_blacklist_outstandingtoken" does not exist
intel_owl_celery_worker | LINE 1: ...ken_blacklist_outstandingtoken"."expires_at" FROM "token_bla...

**Comment by Eshaan7** [comment]
Ah, sorry my mistake. The correct command is `python manage.py migrate`. So it picks up models from the entire application.

**Comment by ibrahim3pf** [comment]
Hello @Eshaan7 

Thank you so much for your support,
unfortunately it didnt work in both ways,
Any way, I will check our maybe the problem is of the OS itself, also I will be checking for the newr released.

Regards.

**Comment by Eshaan7** [comment]
@ibrahim3pf We have released v1.1.0. You can see the changelog download it from here: https://github.com/intelowlproject/IntelOwl/releases/tag/v1.1.0. Follow the [installation steps](https://gist.github.com/ninoseki/83d65b020c86f67f822eb50c56756201) and it should work fine!

I'll be closing this issue now. If you need further support, feel free to re-open this issue or create a new one. :rocket:

**Comment by ibrahim3pf** [comment]
I will check it out @Eshaan7 
Thanks for the support.
